Ahmadreza-tahmasebi/c4.js

```diff
@@ -0,0 +1,26 @@
+"use strict"
+
+function getMaxMin(...args) {
+    if (args.length === 0){
```

Comment:
There is no need to explicitly check the array's length for being zero. Please consider that `0` is a `falsy` value.